json/Item_Stack_ItemBag.txt

```diff
@@ -38565,436 +38565,436 @@
 	},
 	{
 		"assign": "33005769",
 		"jp_text": "☆フィブーラシュプレム箱４",
-		"tr_text": "",
+		"tr_text": "☆Fibla Supreme Box 4",
 		"jp_explain": "以下のアイテムを獲得する。\n「フィブーラシュプレム[Ou]」\n「フィブーラシャルマン雪[Ba]」\n「フィブーラアンブラ」",
-		"tr_explain": ""
+		"tr_explain": "Use to receive the following items:\n[Fibla Supreme [Ou]]\n[Fibla Supreme Snow [Ba]]\n[Fibla Umbrella]"
 	},
 	{
 		"assign": "33005770",
 		"jp_text": "☆フィブーラシュプレム箱５",
-		"tr_text": "",
+		"tr_text": "☆Fibla Supreme Box 5",
 		"jp_explain": "以下のアイテムを獲得する。\n「フィブーラシュプレム[Ou]」\n「フィブーラシャルマン夜[Ba]」\n「フィブーラアンブラ」",
-		"tr_explain": ""
+		"tr_explain": "Use to receive the following items:\n[Fibla Supreme [Ou]]\n[Fibla Supreme Night [Ba]]\n[Fibla Umbrella]"
 	},
 	{
 		"assign": "33005771",
 		"jp_text": "☆フローの筆頭装束Ｆ箱",
-		"tr_text": "",
+		"tr_text": "☆Flo Executive Wear F Box",
 		"jp_explain": "以下のアイテムを獲得する。\n「フローの筆頭装束Ｆ」\n「フローの髪飾りＲ」\n「フラウヘアー」",
-		"tr_explain": ""
+		"tr_explain": "Use to receive the following items:\n[Flo Executive Wear F]\n[Flo Hair Ornament R]\n[Frau Hair]"
 	},
 	{
 		"assign": "33005772",
 		"jp_text": "☆フローの筆頭装束Ｆ雅箱",
-		"tr_text": "",
+		"tr_text": "☆Flo E. Wear F Elegant Box",
 		"jp_explain": "以下のアイテムを獲得する。\n「フローの筆頭装束Ｆ雅」\n「フローの髪飾りＲ」\n「フラウヘアー」",
-		"tr_explain": ""
+		"tr_explain": "Use to receive the following items:\n[Flo Executive Wear F Elegant]\n[Flo Hair Ornament R]\n[Frau Hair]"
 	},
 	{
 		"assign": "33005773",
 		"jp_text": "☆フローの筆頭装束Ｆ雪箱",
-		"tr_text": "",
+		"tr_text": "☆Flo E. Wear F Snow Box",
 		"jp_explain": "以下のアイテムを獲得する。\n「フローの筆頭装束Ｆ雪」\n「フローの髪飾りＲ」\n「フラウヘアー」",
-		"tr_explain": ""
+		"tr_explain": "Use to receive the following items:\n[Flo Executive Wear F Snow]\n[Flo Hair Ornament R]\n[Frau Hair]"
 	},
 	{
 		"assign": "33005774",
 		"jp_text": "☆フローの筆頭装束Ｆ影箱",
-		"tr_text": "",
+		"tr_text": "☆Flo E. Wear F Shadow Box",
 		"jp_explain": "以下のアイテムを獲得する。\n「フローの筆頭装束Ｆ影」\n「フローの髪飾りＲ」\n「フラウヘアー」",
-		"tr_explain": ""
+		"tr_explain": "Use to receive the following items:\n[Flo Executive Wear F Shadow]\n[Flo Hair Ornament R]\n[Frau Hair]"
 	},
 	{
 		"assign": "33005775",
 		"jp_text": "☆フローの筆頭装束Ｆ紅箱",
-		"tr_text": "",
+		"tr_text": "☆Flo E. Wear F Crimson Box",
 		"jp_explain": "以下のアイテムを獲得する。\n「フローの筆頭装束Ｆ紅」\n「フローの髪飾りＲ」\n「フラウヘアー」",
-		"tr_explain": ""
+		"tr_explain": "Use to receive the following items:\n[Flo Executive Wear F Crimson]\n[Flo Hair Ornament R]\n[Frau Hair]"
 	},
 	{
 		"assign": "33005776",
 		"jp_text": "☆ハリエット正礼装　影箱１",
-		"tr_text": "",
+		"tr_text": "☆Harriet Formals Shadow Box 1",
 		"jp_explain": "以下のアイテムを獲得する。\n「ハリエットの正礼装　影[Ou]」\n「ハリエットの正礼装　玄[Ba]」\n「ハリエットのサイハイＢ」他一種",
-		"tr_explain": ""
+		"tr_explain": "Use to receive the following items:\n[Harriet Formals Shadow [Ou]]\n[Harriet Formals Black [Ba]]\n[Harriet Thigh Highs B] +1 other"
 	},
 	{
 		"assign": "33005777",
 		"jp_text": "☆ハリエット正礼装　影箱２",
-		"tr_text": "",
+		"tr_text": "☆Harriet Formals Shadow Box 1",
```

Comment:
should this be "☆Harriet Formals Shadow Box 2"

Reply:
whoops thanks